### PR TITLE
Add Compute Scaling

### PR DESCRIPTION
**Pull request by aayushmahapatra**

### CODE REVIEW

**Review by rekpero — APPROVED** [review]

File: pages/compute/scaling.mdx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# Compute Scaling
+
+## What is compute scaling?
+
+Compute Scaling is the ability to adjust the computational resources allocated to an instance,
+aiming to optimize performance and resource utilization.
+
+We offer the following three types of scaling:
+
+1. No Scaling
```
Comment (rekpero) [inline pages/compute/scaling.mdx:12]: make these grid boxes @aayushmahapatra wdyt?

**Review by rekpero — CHANGES_REQUESTED** [review]
check my comment

File: pages/compute/scaling.mdx (hunk 1 above)
Comment (aayushmahapatra) [reply pages/compute/scaling.mdx:12]: Lemme check

**Review by rekpero — APPROVED** [review]

File: components/scaling.tsx [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import QuickStartCard, { IQuickStartCard } from "./quick-start-card";
+import styles from "../styles/quick.module.css";
+
+const SCALING_LIST = [
+  {
+    path: "/compute/scaling",
+    emoji: "🔴",
```
Comment (rekpero) [inline components/scaling.tsx:7]: use the emoji that is present on the platform. they have some emoji beside the name

**Review by rekpero — CHANGES_REQUESTED** [review]

**Review by rekpero — APPROVED** [review]

**Pull request merged**